python-avd/pyavd/api/interface_descriptions/__init__.py

```diff
@@ -335,6 +335,7 @@ def connected_endpoints_ethernet_interface(self, data: InterfaceDescriptionData)
                     "endpoint": data.peer,
                     "endpoint_port": data.peer_interface,
                     "endpoint_type": data.peer_type,
+                    "endpoint_port_channel_id": data.port_channel_id,
```

Comment:
Let's use the same variable name as the one in `connected_endpoints_port_channel_interface` for consistency

(line 401 of this file)

```suggestion
                    "port_channel_id": data.port_channel_id,
```